audinterface/core/feature.py

```diff
@@ -423,14 +423,10 @@ def _reshape_3d(
     ):
         r"""Reshape to [n_channels, n_features, n_frames]."""
 
+        features = np.array(features)
+        features = np.atleast_1d(features)
+
         if self.process.process_func_is_mono:
-            for channels_features in features:
-                if not isinstance(channels_features, np.ndarray):
-                    raise RuntimeError(
-                        "Features must be a 'np.ndarray', "
-                        f"not '{type(channels_features)}'."
```

Comment:
Seems like we haven't neither mentioned this error in docs not stated that we need to return a numpy array.
I was just wondering about that, but couldn't find an entry in the API docs (we don't have user documentation anyway ;) ).